### DISCUSSION TITLE
Do we still use apt-maven-plugin to generate QClasses from JPA annotations in querydsl 5.0.0 ?

### DISCUSSION THREAD

**Discussion by EzekielaRakotoarijaona** [querydsl/querydsl#3133] [Q&A] [answered]
I try to upgrade my querydsl to 5.0.0 and spring boot 2.6.1 from querydsl 4.4.0 and springboot 2.3.1. When I try to generate QClasses with apt-maven-plugin as it was before, the build does not fail but errors are showing and no classes are generated. Then compilation fails because no QClasses are found.

So my question is do we still use apt-maven-plugin to generate QClasses from JPA annotations in querydsl 5.0.0 or is there some other methods ? I tried some but nothing works.

**Comment by zhanglolo** [comment, marked as answer]
Use as before.

**Comment by hantsy** [comment]
Not required, configure annotationProcessor in maven compiler plugin directly.
